src/apps/service/apps/de/jobs.clj

```diff
@@ -113,8 +113,9 @@
     (submit-standalone-job user submission job)))
 
 (defn- prep-submission
-  [submission]
+  [{:keys [config] :as submission}]
   (assoc submission
+    :config               (:job_config submission config)
```

Comment:
Here is where the new `job_config` key replaces the user's original `config` for DE job submission JSON.